.changelog/11111.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+scheduler: Keep track of system and sysbatch evaluations that fail to place an allocation
```

Comment:
```suggestion
scheduler: Warn users when system and sysbatch evaluations that fail to place an allocation
```

While I don't think this fully encompasses the changes, other bits like marking sysbatch as `dead` immediately are new so haven't changed from a previous release.